live-examples/js-examples/string/string-split.html

```diff
@@ -0,0 +1,7 @@
+<pre>
+<code id="static-js">const str = "The quick brown fox jumped over the lazy dog.";
+
+words = str.split(" ");
```

Comment:
This should be `var words =` not just `words =` (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Grammar_and_types#Declaring_variables).